app/views/users/show.html.erb

```diff
@@ -6,66 +6,19 @@
         <br><br>
         <h3><%= @user.full_name %></h3>
         <h4><%= @user.email %>
-        <!-- <h4>Harvard College</h4>
-        <p><span class="glyphicon glyphicon-user" aria-hidden="true"></span>  Student</p> -->
         <br><br><br>
         <div class="total-points col-centered" data-toggle="tooltip"
              data-placement="bottom" data-trigger="click focus hover"
              title="User score reflects questions authored and votes received on questions">
-          <h2 class="vcenter"><%= @user.karma %></h2>
+          <!--<h2 class="vcenter"><%= @user.karma %></h2>-->
         </div>
         <br><br>
-        <!-- <p>Enrolled in <b></b> classes</p> -->
+        <p><%= @user.questions.group(:qset_id).count('questions.id').length %> question sets</p>
         <p><%= pluralize(@user.questions.count, 'question') %></p>
-        <!-- <p><b>42</b> answers</p> -->
       </div>
     </div>
     <div class="col-sm-9">
-      <!-- <h3>My enrolled classes</h3>
-      <div class="box">
-        <div class="class-bg valign">
-          <h3 class="center class-title ">Stat 110: Intro to Probability</h3>
-        </div>
-        <div class="spacer10"></div>
-        <div class="row center">
-          <div class="col-sm-4"><b>82</b> total questions</div>
-          <div class="col-sm-4"><b>6</b> contributors</div>
-          <div class="col-sm-4"><b>3</b> unanswered</div>
-        </div>
-      </div>
-      <div class="box">
-        <div class="class-bg valign">
-          <h3 class="center class-title ">SLS 20: Intro to Psych</h3>
-        </div>
-        <div class="spacer10"></div>
-        <div class="row center">
-          <div class="col-sm-4"><b>82</b> total questions</div>
-          <div class="col-sm-4"><b>6</b> contributors</div>
-          <div class="col-sm-4"><b>3</b> unanswered</div>
-        </div>
-      </div>
-      <div class="box">
-        <div class="class-bg valign">
-          <h3 class="center class-title ">USW 12: American Encounters</h3>
-        </div>
-        <div class="spacer10"></div>
-        <div class="row center">
-          <div class="col-sm-4"><b>82</b> total questions</div>
-          <div class="col-sm-4"><b>6</b> contributors</div>
-          <div class="col-sm-4"><b>3</b> unanswered</div>
-        </div>
-      </div>
-      <br> -->
-      <h3>Your recent questions</h3>
-      <% if current_user.questions.count < 1 %>
-      <div class="box">
-        <h4 class="center">You haven't generated any questions yet!</h4>
-        <div class="center"><%= link_to "Create one now", new_question_path, class: "btn btn-theme btn-lg" %></div>
-      </div>
-      <% else %>
-      <p>Plug in user questions here</p>
-
-      <% end %>
```

Comment:
Does this go with this PR? Might it be helpful to PR these kinds of refactoring separately, for clarity?